os/board/rtl8730e/src/Makefile

```diff
@@ -0,0 +1,234 @@
+include component/wifi/driver/core/Make.defs
+#include component/ssl/mbedtls_ram_map/rom/Make.defs
+include component/mbed/targets/hal/rtl8730e/Make.defs
+include component/os_dep/Make.defs
+include component/os/tizenrt/Make.defs
+
+ifeq ($(CONFIG_AMEBASMART_WIFI),y)
+include component/wifi/api/Make.defs
+include component/wifi/promisc/Make.defs
+include component/wifi/wpa_supplicant/wpa_supplicant/Make.defs
+include component/soc/amebad2/misc/Make.defs
+endif
+
+ifeq ($(CONFIG_AMEBASMART_BLE),y)
+include component/bluetooth/api/Make.defs
+include component/bluetooth/api/ipc/Make.defs
+include component/bluetooth/os/tizenrt/Make.defs
+include component/bluetooth/example/utils/Make.defs
+ifeq ($(CONFIG_AMEBASMART_BLE_PERIPHERAL),y)
+include component/bluetooth/example/ble_peripheral/Make.defs
+endif
+ifeq ($(CONFIG_AMEBASMART_BLE_CENTRAL),y)
+include component/bluetooth/example/ble_central/Make.defs
+include component/bluetooth/profile/client/Make.defs
+endif
+ifeq ($(CONFIG_AMEBASMART_BLE_SCATTERNET),y)
+include component/bluetooth/example/ble_scatternet/Make.defs
+endif
+endif
+
+ARCH_SRCDIR = 	$(TOPDIR)/arch/$(CONFIG_ARCH)/src
+BOARD_SRCDIR = 	$(TOPDIR)/board
+PROJ_DIR = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/project/realtek_amebaD2_va0_example
+COMPONENT_DIR = $(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component
+DIR_WIFI = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/wifi
+DIR_OSDEP = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/os_dep
+DIR_OS = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/os
+DIR_MBED = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/mbed/targets/hal/rtl8730e
+ifeq ($(CONFIG_AMEBASMART_BLE),y)
+DIR_BT = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/bluetooth
+endif
+
+TARGETDIR = 	$(COMPONENT_DIR)/soc/amebad2
+
+CFLAGS += -I$(DIR)/../include
+CFLAGS += -I$(TOPDIR)/arch/$(CONFIG_ARCH)/src/armv7-a
+CFLAGS += -I$(PROJ_DIR)/inc/inc_ap
+CFLAGS += -I$(TARGETDIR)/bootloader
+CFLAGS += -I$(TARGETDIR)/cmsis
+CFLAGS += -I$(TARGETDIR)/cmsis-dsp/Include
+CFLAGS += -I$(TARGETDIR)/cmsis-dsp/PrivateInclude
+CFLAGS += -I$(TARGETDIR)/fwlib/include
+CFLAGS += -I$(TARGETDIR)/fwlib/ap_core
+CFLAGS += -I$(TARGETDIR)/app/monitor/include
+CFLAGS += -I$(TARGETDIR)/swlib
+CFLAGS += -I$(TARGETDIR)/misc
+CFLAGS += -I$(COMPONENT_DIR)/stdlib
+
+CFLAGS += -I$(COMPONENT_DIR)/wifi
+CFLAGS += -I$(COMPONENT_DIR)/wifi/api
+CFLAGS += -I$(COMPONENT_DIR)/wifi/driver/core
+CFLAGS += -I$(COMPONENT_DIR)/wifi/driver/include
+CFLAGS += -I$(COMPONENT_DIR)/wifi/driver/intf
+CFLAGS += -I$(COMPONENT_DIR)/wifi/inic
+# Not sure if promsic needed
+CFLAGS += -I$(COMPONENT_DIR)/wifi/promisc
+CFLAGS += -I$(COMPONENT_DIR)/wifi/wpa_supplicant/src/crypto
+CFLAGS += -I$(COMPONENT_DIR)/lwip/api
+CFLAGS += -I$(COMPONENT_DIR)/ssl/mbedtls_ram_map/rom
+CFLAGS += -I$(COMPONENT_DIR)/ssl/mbedtls-2.16.9/include
+CFLAGS += -I$(COMPONENT_DIR)/ssl/mbedtls-2.16.9/include/mbedtls
+
+CFLAGS += -I$(COMPONENT_DIR)/os/tizenrt
+CFLAGS += -I$(DIR_MBED)
```

Comment:
I always leave a comment for this. CFLAGS has a length limitation. Please check whether all of them are necessary or not. You can check it next PR.

Reply:
Sure, Mr Chang. We will check all CFLAGS for header include paths and raise another PR.